pkg/tnf/testcases/data/cnf/privilegedpod.go

```diff
@@ -43,10 +43,10 @@ var PrivilegedPodJSON = string(`{
       "name": "HOST_PATH_CHECK",
       "skiptest": true,
        "loop": 0,
-      "command": "oc get pod  %s  -n %s -o json  | jq -r '.spec.hostpath.path'",
+      "command": "oc get pods %s -n %s -o go-template='{{ range .spec.volumes}}{{.hostPath.path}}{{end}}'",
       "action": "allow",
       "expectedstatus": [
-        "NULL_FALSE"
+        "^(<no value>)*$"
       ]
```

Comment:
you're expecting nothing in here for this test to pass ?
that seems a weird test. I though you need to find volumes mounted over a network

Reply:
After looking at the initial test again, this was really the intent, the implementation was just flawed. The function I tested last week was also looping through the PVC and PVs, but is beyond
 what we intend to test before and for this we will need a template file.

Reply:
same question, why not find host-path mounted